### PR TITLE
Add remove edit

### PR DESCRIPTION
**Pull request by zieeco**
## To Do List: Add_remove
**In this milestone, I did the following**
- Added a `status update` file in the `src` folder 
- Used the `npm run build` command to generate the files in the `dist folder`
- Ensured there were no linters error
- Added some classes in the `style.css` file
- **Added the `index ` parameter to the `saveToDos()` in the  `index.js` file according to the required changes from the previous code review..**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -144,6 +144,10 @@ div.task-container ul li div {
   border-bottom: 1px solid #b8b8b8;
 }
 
+.line-through {
+  text-decoration: line-through;
+}
+
```
Comment (BrianSammit) [inline src/style.css:156]: ## OPTIONAL  ♻️

- [ ] Add the property `cursor: pointer` to the icons, this will help with the UI. 
😄 🔧

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 🔴 🛑
Hello @zieeco, great job you are doing so far in this project. 💯
TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- Great job with the add, update and delete actions. 

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️

- [ ] You are missing the property index in the tasks. kindly add this property as one of the project requirements. 

Capture

Remember that the index value must be the length of the array. 

Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

tenor-1



**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/style.css (hunk 1 above)
Comment (zieeco) [reply src/style.css:156]: Fixed and thanks for your review

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @zieeco ,  

Index property is setup properly and is being updated every time an idem is deleted! Good Job! 🎉💯🌟 
Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**